kahuna/public/js/components/gr-image-metadata/gr-image-metadata.js

```diff
@@ -40,6 +40,28 @@ module.controller('grImageMetadataCtrl', [
     ctrl.metadata = ctrl.image.data.metadata;
     ctrl.identifiers = ctrl.image.data.identifiers;
 
+    mediaApi.fileMetadata(ctrl.image).then(resource => {
+      return resource.data;
+    }).then(fileMetadata => {
+      const groupedConfigsByDirectory = window._clientConfig.fileMetadataConfig.reduce((r, a) => {
+        r[a.directory] = [...r[a.directory] || [], a];
+        return r;
+      }, {});
+
+      let object = {};
+
+      Object.keys(groupedConfigsByDirectory).forEach(directory => {
+        groupedConfigsByDirectory[directory]
+          .filter(config => config.visible)
+          .filter(fileMetadataConfig => fileMetadata[directory][fileMetadataConfig.tag] !== undefined)
+          .map(fileMetadataConfig => {
+            object[fileMetadataConfig.alias] = fileMetadata[directory][fileMetadataConfig.tag];
+          });
+      });
```

Comment:
A cleaner pattern for this (once flattened to remove the directories) uses `Object.fromEntries`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/fromEntries#object_transformations

Reply:
Removed this bit that's now irrelevant. The visible fileMetadata fields are accessible within `ctrl.image`